$NetBSD: patch-miniircd,v 1.1 2020/08/21 16:44:11 leot Exp $

Properly format default parameter in argparse help messages (fixes
`--help' option).

Backport from upstream commit dd04e1352102f5d30920334e71f4b376b75c4f17.

--- miniircd.orig	2020-07-16 11:48:28.000000000 +0000
+++ miniircd
@@ -1087,7 +1087,7 @@ def main(argv: Sequence[str]) -> None:
         metavar="X",
         default=10,
         type=int,
-        help="keep X log files; default: %default",
+        help="keep X log files; default: %(default)s",
     )
     ap.add_argument("--log-file", metavar="X", help="store log in file X")
     ap.add_argument(
@@ -1095,7 +1095,7 @@ def main(argv: Sequence[str]) -> None:
         metavar="X",
         default=10,
         type=int,
-        help="set maximum log file size to X MiB; default: %default MiB",
+        help="set maximum log file size to X MiB; default: %(default)s MiB",
     )
     ap.add_argument(
         "--motd", metavar="X", help="display file X as message of the day"
